### DISCUSSION TITLE
Unable to view NVME Disk in DSM 7.2-64570 Update 1 DS918+

### DISCUSSION THREAD

**Discussion by popovici-gabriel** [007revad/Synology_HDD_db#190] [Q&A] [answered]
DS918+
Using TinyCore https://github.com/pocopico/tinycore-redpill 
Setup on BareMetal intel Core i3 
ASRock Rack C236 WSI Mini ITX Server Motherboard LGA 1151 Intel C236 
Sabrent 512GB Rocket NVMe PCIe M.2 2280 Internal SSD High Performance Solid State Drive (SB-ROCKET-512) 
on NVMe to PCIe Adapter x16, RIITOP M.2 NVMe SSD to PCI-e 3.0 x16 Card with Heatsink Support M Key M.2 NVMe SSD 2230, 2242, 2260, 2280 
DSM 7.2-64570 Update1 

sudo nvme list
Node             SN                   Model                                    Namespace Usage                      Format           FW Rev
---------------- -------------------- ---------------------------------------- --------- -------------------------- ---------------- --------
/dev/nvme0n1     23CD0714099800072660 Sabrent                                  1         512.11  GB / 512.11  GB    512   B +  0 B   RKT343.4
/dev/nvme0n1p1   23CD0714099800072660 Sabrent                                  1         512.11  GB / 512.11  GB    512   B +  0 B   RKT343.4
/dev/nvme0n1p2   23CD0714099800072660 Sabrent                                  1         512.11  GB / 512.11  GB    512   B +  0 B   RKT343.4
/dev/nvme0n1p3   23CD0714099800072660 Sabrent                                  1         512.11  GB / 512.11  GB    512   B +  0 B   RKT343.4

The NVME storage is not available in the DSM Storage. 

Any idea what to do next ??? 

![image](https://github.com/007revad/Synology_HDD_db/assets/11446759/fd51ed92-c646-4202-aa63-bc64f2a5bd2b)

**Comment by popovici-gabriel** [comment]
Added a similar question on https://xpenology.com/forum/topic/69992-unable-to-view-nvme-disk-in-dsm-72-64570-update-1-ds918/#comment-454126 maybe someone has any input. 

Thanks 
Gabe

**Comment by 007revad** [comment]
With a DS918+ you need the nvme-cache addon from https://github.com/PeterSuh-Q3/tcrp-addons to change 0000:00:13.1 to match your PCIe card's 0000:00:01.0

**Comment by popovici-gabriel** [comment]
Thanks for the reply. Would u be kind to explain me how to run the script https://github.com/PeterSuh-Q3/tcrp-addons/blob/main/nvme-cache/installondsm.sh 

How do I change from  0000:00:13.1 to 0000:00:01.0 ? 
Thanks 
Gabe

**Comment by 007revad** [comment, marked as answer]
You don't need that install script. And I've actually found a better script anyway.

Click on the little download button here: https://github.com/pocopico/tcrp-addons/blob/main/nvme-cache/src/nvme-cache.sh 

You can run a script in task scheduler. https://github.com/007revad/Synology_DSM_reinstall/blob/main/how_to_run_from_scheduler.md Replace the path to script and the script name to match where you downloaded the sh file and it's name.
